ls_0/my_goals.json

```diff
@@ -1,7 +1,7 @@
-  {
+ [  {
```

Comment:
File should be in a json format. So it couldn`t array-like.
To apply some changes you can just change something in the text, but go back to json-format like:
`{
    "key": "value"
}`

Reply:
Allright , 
I checked it on json checker and it worked there .
No big deal i gonna remove it